.flowconfig

```diff
@@ -13,6 +13,7 @@
 .*/test/local-storage/local-storage-utils.test.js
 .*/test/tracker-component.test.js
 .*/test/.*
+<PROJECT_ROOT>/webpack.dev.mjs
```

Comment:
why we need this?

Reply:
Ignore type checking in webpack configuration